### PR TITLE
fix: correct sign of offset and negation of expression

### PR DESCRIPTION
**Pull request by msvticket**
Fix to make it work like intended in #920

### CODE REVIEW

**Comment by msvticket** [comment]
You are welcome to check as well @Skisocks

**Review by tomhobson — APPROVED** [review]

**Review by Skisocks — APPROVED** [review]

File: pkg/cmd/helmfile/status/status_test.go [hunk 1]
```diff
@@ -27,7 +27,7 @@ const (
 
 var fullRepoName = filepath.Join(repoOwner, repoName)
 
-func TestHemlfileStatus(t *testing.T) {
+func TestHemlfileStatus_WithEarlyCutoff(t *testing.T) {
```
Comment (tomhobson) [inline pkg/cmd/helmfile/status/status_test.go:30]: Can you spell helmfile right -- it's frustrating me :)

**Review by tomhobson — CHANGES_REQUESTED** [review]

File: pkg/cmd/helmfile/status/status_test.go (hunk 1 above)
Comment (tomhobson) [reply pkg/cmd/helmfile/status/status_test.go:30]: damn missed the merge window

**Pull request merged**